src/index.html

```diff
@@ -7,8 +7,41 @@
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>Moyo header</title>
   <link rel="stylesheet" href="./style.css">
+  <link href="https://fonts.googleapis.com/css?family=Roboto:500&display=swap" rel="stylesheet">
 </head>
 <body>
-  <h1>Moyo header</h1>
+  <header class="header">
+    <div class="logo">
+      <a href="#"><img src="./images/logo.png" alt="logo"></a>
```

Comment:
One tag - one line. Use new lines for nested tags. Don't write nested tags in one line.